### PR TITLE
Add a small footer

### PR DESCRIPTION
**Pull request by vnbaaij**
With the new summary/details view, text seems to disappear into the abys at the bottom of the window. I feel adding a small footer helps to make that less apparent. I also remove the bottom padding from various pages to make the text/splitter stop at the footer and not above it

Text displayed in the footer can of course be adjusted.
Before:
image

After:
image

### CODE REVIEW

**Review by smitpatel — APPROVED** [review]

**Comment by JamesNK** [comment]
I don't think fixed footers look very good. I prefer the before result.

File: src/Aspire.Dashboard/Components/Layout/MainLayout.razor [hunk 1]
```diff
@@ -28,6 +28,9 @@
     <FluentBodyContent Class="custom-body-content">
         @Body
     </FluentBodyContent>
+    <FluentFooter>
```
Comment (IEvangelist) [inline src/Aspire.Dashboard/Components/Layout/MainLayout.razor:31]: Could we add another footer message that is left aligned? With the following:

`Powered by @(System.Runtime.InteropServices.RuntimeInformation.FrameworkDescription)`

Comment (vnbaaij) [reply src/Aspire.Dashboard/Components/Layout/MainLayout.razor:31]: Done. Also removed the padding on the right so the scrollbar attaches to the side of the window.
 
image

**Comment by smitpatel** [comment]
Let's wait for @leslierichardson95 input here.

**Comment by leslierichardson95** [comment]
I'm slightly conflicted here.  My main concern is that having a fixed footer could convey the notion that there's nothing else in the list up top but I also understand the original concern.  As a possible compromise, what about having a different colored (i.e. white in this case) outline around or at least above the summary / details view so there's a more solid distinction between the two windows?

**Comment by smitpatel** [comment]
Another thing is there a way to stop the table to stop rendering partial row? Like if you don't have enough height to show, just hide whole row?

**Comment by JamesNK** [comment]
My POV is that fixed footers feel like web design from a bygone area. I can't think of a website/app off the top of my head that has one.

**Comment by danmoseley** [comment]
I'm going to close this and suggest opening an issue to get consensus if you wish @vnbaaij

**Comment by vnbaaij** [comment]
Nah, all good with closing this. It was just an attempt to try to improve aesthetics a bit . But I'm not a designer so should probably not try to act like one 😀

**Comment by danmoseley** [comment]
@vnbaaij I think your design input is valuable, please do keep offering input! But sometimes a thing is better started as an issue to get consensus.

**Pull request closed without merging**